src/Query/Filter.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Query;
+
+use Symfony\Component\HttpFoundation\Request;
+
+final class Filter
+{
+    private int $offset = 0;
+    private int $limit = 20;
+    private ?string $searchTerm;
+
+    public function __construct(int $offset = 0, int $limit = 20, ?string $searchTerm = null)
+    {
+        if ($offset > 0) {
+            $this->offset = $offset;
+        }
+
+        if ($limit >= 0) {
```

Comment:
We should add max limit, to prevent return 1k+ rows in one reqeust. I think `100` should be fine :thinking: